homeassistant/components/cloud/client.py

```diff
@@ -106,15 +104,6 @@ def alexa_config(self) -> alexa_config.AlexaConfig:
 
         return self._google_config
 
-    async def async_initialize(self, cloud: Cloud) -> None:
-        """Initialize the client."""
-        self.cloud = cloud
-
-        if not self.cloud.is_logged_in:
-            return
-
-        await self.logged_in()
```

Comment:
This is not allow on an interface class design pattern. I restruct the backend library to avoid this construct.